src/components/Sorted.jsx

```diff
@@ -0,0 +1,50 @@
+import { useSearchParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { fetchSortOrder } from "../api";
+import ArticleCard from "./ArticleCard";
+
+export default function Sorted () {
```

Comment:
This isn't a great name for a React Component
Try to choose a name that represents what UI the component produces

In this case it's producing a list of articles!